hbt/production/btag.py

```diff
@@ -93,24 +93,25 @@ def _normalized_btag_weights_init(self: Producer) -> None:
 @_normalized_btag_weights.requires
 def _normalized_btag_weights_requires(self: Producer, reqs: dict) -> None:
     from columnflow.tasks.selection import MergeSelectionStats
-    reqs["selection_stats"] = MergeSelectionStats.req(
+    reqs["selection_stats"] = MergeSelectionStats.req_different_branching(
         self.task,
-        tree_index=0,
-        branch=-1,
-        _exclude=MergeSelectionStats.exclude_params_forest_merge,
+        branch=-1 if self.task.is_workflow() else 0,
     )
 
 
 @_normalized_btag_weights.setup
 def _normalized_btag_weights_setup(self: Producer, reqs: dict, inputs: dict, reader_targets: InsertableDict) -> None:
+    # BUG in prod3: some stats fields were missing so skip them for now
+    return
```

Comment:
which stats fields are we talking about?

Reply:
This was one of the hacks I was talking about. I removed it again from the branch.